omero/sysadmins/version-requirements.txt

```diff
@@ -1229,9 +1229,9 @@ Ice
       - from June 2015
       - to TBA
       - Unsupported
-      - Unsupported
+      - Supported
       - Recommended
-      - :zerocforum:`Ice 3.6.0 <announcements/6631-ice-3-6-0-and-ice-touch-3-6-0-released>`
```

Comment:
Sorry I misunderstood the purpose of the column
I will add back 3.6 and also add 3.6.1